### DISCUSSION TITLE
Variables documentation

### DISCUSSION THREAD

**Discussion by stoeps13** [HCL-TECH-SOFTWARE/connections-automation#196] [General]
Thanks for the new documentation about the used variables.

It looks like the last lines are duplicates, or not sorted into the right topic: https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/42b7ad04889cc27f5bc0e58fa7f853541df9326a/documentation/VARIABLES.md?plain=1#L360-L393 

Regards
Christoph

**Comment by nitinjagjivan** [comment]
Hi @stoeps13,
Highlighted variables are needed in the 'cleanup variables' section as we are using it while running cleanup playbooks.

For e.g. db2_instance_fenced_homedir
This variable is used while running 'playbooks/hcl/cleanup/cleanup-db2.yml' playbook. We need this path to understand what is the fenced user home directory. So that this path can be used to clean up IBM db2 as a part of db2 cleanup job.
You can see this variable listed in 'DB2 Variables' section as well as it is used while installing IBM DB2. We need to duplicate it as it will be handy for the users who just want to run cleanup job.

**Comment by stoeps13** [comment]
Thanks @nitinjagjivan for the heads up,
I don't like the idea to edit the variables file to uninstall and then switch it back. I do similar operations with the usage of tags. So calling `ansible-playbook --tags uninstall` is uninstalling and tags=install is set to default. Then the variables file can stay the same.
I would only document the variables which are necessary for uninstallation, in my workflow I would install first, then add the 4 or 5 uninstall vars and uninstall. So reading about more variables (which should be set earlier for installation) can be overwhelming and lead to errors.
If you install without setting these variables, they use the defaults. If not you have already set them.

**Comment by nitinjagjivan** [comment]
Thanks for the suggestion. Our team will think about this approach and will decide considering all other priority items.
